geomstats/geometry/stratified/graph_space.py

```diff
@@ -185,14 +284,14 @@ def __init__(self, n_nodes, total_space=None):
 
     @_pad_with_zeros((1, "graphs"))
     def belongs(self, graphs, atol=gs.atol):
-        r"""Check if the matrix is an adjacency matrix.
+        r"""Check if the point belongs to the space.
 
         The adjacency matrix should be associated to the
         graph with n nodes.
 
         Parameters
         ----------
-        graphs : list of Graph or array-like, shape=[..., n, n].
+        graphs : list of GraphPoint or array-like, shape=[..., n, n].
```

Comment:
`n` was called `n_nodes` in previous docstrings. Let's uniformize the notations, using n_nodes everywhere? (since this is the name of the attribute in the class)